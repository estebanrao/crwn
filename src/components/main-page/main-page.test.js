import { shallow } from 'enzyme';
import React from 'react';
import MainPage from './main-page.component';

it('should render MainPage component', () => {
  expect(shallow(<MainPage />)).toMatchSnapshot();
});
