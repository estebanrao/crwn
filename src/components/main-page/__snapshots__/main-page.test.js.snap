// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MainPage component 1`] = `ShallowWrapper {}`;
